app/javascript/flavours/glitch/features/status/components/detailed_status.jsx

```diff
@@ -132,6 +132,17 @@ class DetailedStatus extends ImmutablePureComponent {
       return null;
     }
 
+    const fields = status.getIn(['account', 'fields']);
+    let pronouns = null;
+
+    if (fields) {
+      fields.forEach(field => {
+        if (field.get('name') === 'pronouns') {
```

Comment:
you would probably wanna add variations to this, eg "professional nouns" and other variations people like to use on Fedi. You could take a look at Moshidon, they have that problem largely figured out, but I don't have a link to *where exactly* that is in the src

Reply:
i’m not seeing anywhere where they handle labels like “professional nouns” or other variations but fwiw the logic is here: https://github.com/LucasGGamerM/moshidon/blob/43afbb7523bde9072aed8f7abd0f467bc17a1c38/mastodon/src/main/java/org/joinmastodon/android/ui/utils/UiUtils.java#L1919

as far as i can see it looks like it handles the current “pronouns” label in whatever language you have the app in, along with english, and nothing else. but for the value of the pronouns there’s a lot more logic for things like urls